packages/lsp-server/package/package.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
should not this file sit directly under packages/lsp-server? why the additional `package` sub folder?